ldregistry/templates/nav/_menu-advanced.vm

```diff
@@ -4,7 +4,7 @@
     <a href="#" class="dropdown-toggle" data-toggle="dropdown">$msg['nav.advanced.label']<b class="caret"></b></a>
     <ul class="dropdown-menu">
         #if( ! $subject.isAuthenticated() )
-            <li><a href="$absoluteRoot/ui/login?return=$absoluteRoot/$uri">$msg['login.button']</a></li>
+            <li><a href="$absoluteRoot/ui/login?return=/$uri">$msg['login.button']</a></li>
```

Comment:
This will not work if the registry is deployed at a non-root context.

`$absoluteRoot` is supposed to be `/` in the case where the registry is deployed at root. If that's not working then that's a separate issue and prefer to fix or work around that rather than drop support for context paths.